layouts/partials/head_scripts/datadog_monitoring.html

```diff
@@ -1,19 +1,20 @@
+<!-- Datadog Monitoring -->
 <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-rum-us.js"></script>
 <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-logs-us.js"></script>
 <script>
 
   // Only initializing RUM if on preview or live env
-{{ if (or (eq $.Site.Params.environment "live") (eq $.Site.Params.environment "preview")) }}
-    DD_RUM.init({
-      clientToken: '{{ $.Site.Params.datadog.clientToken }}',
-      applicationId: '{{ $.Site.Params.datadog.applicationId }}',
-    });
-{{ end }}
+  { { if (or(eq $.Site.Params.environment "live")(eq $.Site.Params.environment "preview")) } }
+  DD_RUM.init({
+    clientToken: '{{ $.Site.Params.datadog.clientToken }}',
+    applicationId: '{{ $.Site.Params.datadog.applicationId }}',
+  });
+  { { end } }
```

Comment:
In the page source for the homepage i see references to `$.Site`
I think these extra spaces in the curly braces are causing it to output incorrectly? it might need to be `{{` instead of `{ {`

Reply:
indeed, i just removed the logic since now all DD monitoring is disabled on dev env.

Reply:
This seems to be resolved already